src/ipyvizzu/chart.py

```diff
@@ -100,20 +99,36 @@ def _merge_animations(animations):
         return merger
 
     def feature(self, name, enabled):
-        display_javascript(
+        self._display(
             DisplayTemplate.FEATURE.format(
                 chart_id=self._chart_id,
                 **Feature(name, enabled).dump(),
-            ),
-            raw=True,
+            )
         )
 
     def store(self) -> Snapshot:
         snapshot_id = uuid.uuid4().hex[:7]
-        display_javascript(
+        self._display(
             DisplayTemplate.STORE.format(
                 chart_id=self._chart_id, **Store(snapshot_id).dump()
-            ),
-            raw=True,
+            )
         )
         return Snapshot(snapshot_id)
+
+    def show(self):
+        if self._display_target != DisplayTarget.MANUAL:
+            raise NotImplementedError(
+                f'chart.show() can be used with display="{DisplayTarget.MANUAL}"'
+            )
+        display_javascript(
+            "\n".join(self._calls),
+            raw=True,
+        )
```

Comment:
Clearing self._calls here?

Reply:
I modified it.
If use  _display or show methods after the show function called once, it throws an exception.